Fix obtained from https://github.com/libgd/libgd/commit/f4bc1f5c26925548662946ed7cfa473c190a104a

It should be available in libgd 2.3.4 (I suppose there is a typo) according to
https://githubmemory.com/repo/libgd/libgd/issues/777

The change is needed in order to compile php

From f4bc1f5c26925548662946ed7cfa473c190a104a Mon Sep 17 00:00:00 2001
From: Remi Collet <remi@remirepo.net>
Date: Mon, 13 Sep 2021 14:57:52 +0200
Subject: [PATCH] Revert "Fix #318, these macros are not used as planed, we
 have separate functions for each"

This reverts commit bdc281eadb1d58d5c0c7bbc1125ee4674256df08.
---
 src/gd.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/gd.h b/src/gd.h
index 30560395..1ad9e637 100644
--- src/gd.h
+++ src/gd.h
@@ -1604,6 +1604,11 @@ BGD_DECLARE(void) gdImageFlipHorizontal(gdImagePtr im);
 BGD_DECLARE(void) gdImageFlipVertical(gdImagePtr im);
 BGD_DECLARE(void) gdImageFlipBoth(gdImagePtr im);
 
+#define GD_FLIP_HORINZONTAL 1 /* typo, kept for BC */
+#define GD_FLIP_HORIZONTAL 1
+#define GD_FLIP_VERTICAL 2
+#define GD_FLIP_BOTH 3
+
 /**
  * Group: Crop
  *
